### PR TITLE
Update build.gradle

### PR DESCRIPTION
**Pull request by philipheinser**

### CODE REVIEW

**Comment by edvinasbartkus** [comment]
👍

**Comment by npomfret** [comment]
Thanks for this.  What effect will it have?

File: android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotification.java [hunk 1]
```diff
@@ -68,6 +68,11 @@ public void onNewIntent(Intent intent) {
             bundle.putBoolean("foreground", false);
             intent.putExtra("notification", bundle);
             notifyNotification(bundle);
+        }else if ( intent.getExtras() != null ) {
+            Bundle bundle = intent.getExtras();
+            bundle.putBoolean("foreground", false);
+            intent.putExtra("notification", bundle);
+            notifyNotification(bundle);
         }
     }
```
Comment (npomfret) [inline android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotification.java]: What's this for?

Comment (philipheinser) [inline android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotification.java]: this fixed a problem when you open the app from a click on a notification the notification wasn't available in js

Comment (npomfret) [inline android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotification.java]: Ok, thanks.  Would you mind resolving  the conflicts and I'll look at adding it in.  Also perhaps rename it, the description isn't accurate.  And could you back out the gradle change (and submit it as a separate PR maybe)?

Comment (npomfret) [inline android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotification.java]: Can you please tell me how to test this, I'm unable to create a notification that when I tap it in the notification centre it hits this new 'else' block.

Comment (npomfret) [inline android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotification.java]: I'd be happy to merge this, but I can't just test it.  What do I need to do in order to invoke the new code?

**Comment by npomfret** [comment]
Hi - apologies for the delay, and thank you for submitting this change.  Struggling to find time to look at PR's on this project. 

Can you please make sure you describe the motivation for this change, and crucially **what is the expected change in behaviour**?  How could someone test it?

I would suggest you ask the authors of other outstanding PR's to test your change in a private fork.  If you can get the thumbs up from a couple of other developers then I'd be happy to merge it.  In exchange you could offer to test theirs maybe..?  That would really help get this project moving.

**Comment by Gp2mv3** [comment]
I'll close this PR to clean a bit the repo, feel free to open it back if you think it was closed by mistake.

**Pull request closed without merging**